deploy/operator/apply/01_FabricGatewayCRD.yaml

```diff
@@ -113,6 +113,32 @@ spec:
                     description: |
                       A header which an XHR request should be allowed to use.
                     example: Authorization
+            x-fabric-employee-access:
+              type: object
+              description: |
+                The `user-list` should only be populated if `type` is set to `allow_list`. If `type` is not
+                present, it will be defaulted to `scoped_access`
+              properties:
+                user-list:
+                  type: array
+                  items:
+                    type: string
+                    description: |-
+                      UID of a user who should bypass scope checks, since user tokens can't have scopes.
+                type:
+                  type: string
+                  enum:
+                    - allow_list
+                    - allow_all
+                    - deny_all
+                    - scoped_access
```

Comment:
I like that you added in `scoped_access`. It helps describe the behaviour quite a bit